bin/internal/update_engine_version.sh

```diff
@@ -14,10 +14,24 @@
 
 set -e
 
+# Allow overriding the intended engine version via FLUTTER_ENGINE_VERSION.
+#
+# This is for systems, such as Github Actions, where we know ahead of time the
+# base-ref we want to use (to download the engine binaries and avoid trying
+# to compute one below).
+#
+# This environment variable is EXPERIMENTAL. If you are not on the Flutter infra
+# team, this code path might be removed at anytime and cease functioning. Please
+# file an issue if you have workflow needs.
+if [ -n "${FLUTTER_ENGINE_VERSION}" ]; then
+  ENGINE_VERSION="${FLUTTER_ENGINE_VERSION}"
+  echo "[Unstable] Override: Setting engine SHA to $ENGINE_VERSION" 1>&2
```

Comment:
Note that `engine.version` override is also happening on dart/flutter hh: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester.py#104.

Please keep dart infra team in the loop regarding changes around mechanism that allows to override engine.version

Reply:
What is there to inform? The Dart team isn't using `FLUTTER_ENGINE_VERSION` (right?) and this is an internal experiment we're trying not to publicize intentionally.